src/EventConsumerService.cpp

```diff
@@ -51,11 +51,29 @@ void EventConsumerService::processAccessoryEvent(VlcbMessage *msg, unsigned int
   }
 }
 
+void EventConsumerService::process(UserInterface::RequestedAction requestedAction)
+{
+  if (coeService && (coeService->available()))
+  {
+    // DEBUG_SERIAL << ">Getting COE Message " << endl;
+    VlcbMessage *msg = coeService->get();
```

Comment:
First I thought the messages from the COE Service buffer would go to Controller for normal processing of messages. But you are right here in that we are only interested in event messages.
Just a bit confusing that the process() call is retrieving messages and handling them where we have handleMessage for that.

Reply:
Yes, I originally sent it to Controller but then decided that it was just creating a complex route.